pages/governance/functionalities.md

```diff
@@ -76,12 +76,22 @@ Liquidity Tiers group markets of similar risk into standardized risk parameters.
 
 Current Liquidity Tiers include: 
 
-| ID | Name | initial margin fraction | maintenance fraction (what fraction MMF is of IMF) | base position notional | impact notional |  maintenance margin fraction (as is) | impact notional (as is) |
-| --- | --- | --- | --- | --- | --- | --- | --- |
-| 0 | Large-Cap | 0.05 | 0.6 | 1_000_000 USDC | 500 USDC / IM | 0.03 | 10_000 USDC |
-| 1 | Mid-Cap | 0.1 | 0.5 | 250_000 USDC | 500 USDC / IM | 0.05 | 5_000 USDC |
-| 2 | Long-Tail | 0.2 | 0.5 | 100_000 USDC | 500 USDC / IM | 0.1 | 2_500 USDC |
-| 3 | Safety | 1 | .02 | 1_000 USDC | 2500 USDC / IM | 0.2 | 2_500 USDC |
+| ID | Name | initial margin fraction | maintenance fraction (what fraction MMF is of IMF) | base position notional | impact notional |  maintenance margin fraction (as is) | impact notional (as is) | Lower Cap (USDC Millions) | Upper Cap (USDC Millions) |
+| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
+| 0 | Large-Cap | 0.05 | 0.6 | 1_000_000 USDC | 500 USDC / IM | 0.03 | 10_000 USDC | None | None |
+| 1 | Mid-Cap | 0.1 | 0.5 | 250_000 USDC | 500 USDC / IM | 0.05 | 5_000 USDC | 15 | 25 |
+| 2 | Long-Tail | 0.2 | 0.5 | 100_000 USDC | 500 USDC / IM | 0.1 | 2_500 USDC | 2.5 | 5 |
+| 3 | Safety | 1 | .02 | 1_000 USDC | 2500 USDC / IM | 0.2 | 2_500 USDC | 0.5 | 1 |
+
+- Each market has a `Lower Cap` and `Upper Cap` denominated in USDC.
+- Each market already has a `Base IMF`.
+- At any point in time, for each market:
+    - Define
+        - `Open Notional = Open Interest * Oracle Price`
+        - `Scaling Factor = (Open Notional - Lower Cap) / (Upper Cap - Lower Cap)`
+        - `IMF Increase = Scaling Factor * (1 - Base IMF)`
+    - Then a market’s `Effective IMF = Min(Base IMF + Max(IMF Increase, 0), 1.0)`
+- The effective IMF is the base IMF while the OI < lower cap, and increases linearly until OI = Upper Cap, at which point the IMF stays at 1.0 (requiring 1:1 collateral for trading)
```

Comment:
based on the equation shouldn't this be `The effective IMF is the base IMF while the Open Notional < lower cap` ?
Because Open Interest can be very low but if oracle price is extremely high it will still influence the effective IMF